src/PowerShell/Microsoft.WinGet.Client.Cmdlets/Properties/AssemblyInfo.cs

```diff
@@ -10,7 +10,7 @@
 
 // Forcibly set the target and supported platforms due to the internal build setup.
 // Keep in sync with project versions.
-[assembly: TargetPlatform("Windows10.0.22000.0")]
+[assembly: TargetPlatform("Windows10.0.26100.0")]

```

Comment:
I feel like there was a reason that I didn't update that... but now I'm not remembering it.